casepro/statistics/signals.py

```diff
@@ -38,6 +41,29 @@ def record_new_outgoing(sender, instance, created, **kwargs):
         if instance.partner:
             DailyCount.record_item(day, DailyCount.TYPE_REPLIES, partner)
 
+        if case:
+            # count the very first response on an org level
+            if instance == case.outgoing_messages.earliest('created_on'):
```

Comment:
Cool, that sounds like a reasonable compromise. I'll start on the PR for ignoring self-assigned cases and the rename.
re mege conflicts - I pushed the fix for that this morning.